pkg/tests/observability_retention_test.go

```diff
@@ -52,14 +56,10 @@ var _ = Describe("Observability:", func() {
 				}
 			}
 		}
-		idmk, err := strconv.Atoi(deleteDelay[:len(deleteDelay)-1])
-		if err != nil {
-			panic(err.Error())
-		}
-		ignoreDeletionMarksDelay = strconv.Itoa(idmk/2) + deleteDelay[len(deleteDelay)-1:]
 	})
 
-	It("[P2][Sev2][Observability][Stable] Check compact args: --delete-delay="+deleteDelay+" (retention/g0)", func() {
+	It("[P2][Sev2][Observability][Stable] Check compact args:", func() {
+		By("--delete-delay=" + deleteDelay + " (retention/g0)")
```

Comment:
if move `(retention/g0)` from `It` to `By`, it breaks the `focus` feature. Our cases are designed to be run by `--focus`.

Reply:
updated